### PR TITLE
Update cloud provider docs

### LINKED ISSUE #535: Cloud Provider docs out of date
https://docs.radapp.dev/operations/providers/#configure-a-cloud-provider

- Should switch to the new `--set` command
- Should use `rad init` and `rad credential register` where applicable

[AB#7340](https://dev.azure.com/azure-octo/e61041b4-555f-47ae-95b2-4f8ab480ea57/_workitems/edit/7340)

### PR DESCRIPTION
**Pull request by Reshrahim**
Thank you for helping make the Radius documentation better!

**Please follow this checklist before submitting:**

- [ ] [Read the contribution guide](https://docs.radapp.dev/contributing/docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description
Remove 'rad env init' reference and update the reinstall commands for adding a cloud provider to existing env

### CODE REVIEW

File: docs/content/operations/providers/index.md [hunk 1]
```diff
@@ -120,9 +132,9 @@ The AWS provider allows you to deploy and connect to AWS resources from a Radius
 
 #### Add a cloud provider when initializing an environment
 
-1. Initialize a new [environment]({{< ref environments >}}) with `rad env init kubernetes -i`
-1. Enter "n" to add an Azure cloud provider
-1. Enter "y" to add an AWS cloud provider
+1. Initialize a new [environment]({{< ref environments >}}) with `rad init`
+1. Select the kubernetes cluster to install Radius. Enter an environment name and namespace to deploy the apps into. Note that this namespace is used for application deployments.
+1. Select "yes" to add a cloud provider and select Azure as the cloud provider
```
Comment (Reshrahim) [inline docs/content/operations/providers/index.md]:
```suggestion
1. Select "yes" to add a cloud provider and select AWS as the cloud provider
```

File: docs/content/operations/providers/index.md [hunk 2]
```diff
@@ -36,21 +36,33 @@ The Azure provider allows you to deploy and connect to Azure resources from a se
...
+   ```
+   Replace it with your service principal appId, password and tenant id
+
+1. Update your environment with your Azure subscription and resource group
+   ```bash
+   rad env update <myenv> --azure-subscription-id <subscriptionid> --azure-resource-group <resourcegroupname> 
+   ```
+1. Deploy your app and any included Azure resources with `rad deploy`
 
 #### Add a cloud provider to an AKS cluster using AAD pod identity
```
Comment (AaronCrawfis) [inline docs/content/operations/providers/index.md]: Let's actually remove this section altogether, it's no longer applicable

Comment (Reshrahim) [reply docs/content/operations/providers/index.md]: removed

File: docs/content/operations/providers/index.md [hunk 3]
```diff
@@ -36,76 +36,35 @@ The Azure provider allows you to deploy and connect to Azure resources from a se
 
 #### Add a cloud provider when initializing an environment
 
-1. Initialize a new [environment]({{< ref environments >}}) with `rad env init kubernetes -i`
-1. Enter "y" to add an Azure cloud provider
+1. Initialize a new [environment]({{< ref environments >}}) with `rad init`
+1. Select the kubernetes cluster to install Radius. Enter an environment name and namespace to deploy the apps into. Note that this namespace is used for application deployments.
```
Comment (AaronCrawfis) [inline docs/content/operations/providers/index.md]:
```suggestion
1. Select the Kubernetes cluster to install Radius into. Enter an environment name and base Kubernetes namespace to deploy the apps into.
```

File: docs/content/operations/providers/index.md [hunk 4]
```diff
@@ -36,76 +36,35 @@ The Azure provider allows you to deploy and connect to Azure resources from a se
...
 #### Add a cloud provider to an existing environment
 
-1. Reinstall the control plane with the cloud provider via `rad install kubernetes --reinstall -i`
-   - If using a Codespace or k3s, append `--public-endpoint-override "localhost:8081"` to the command
-   - If using Kind, append `--public-endpoint-override "localhost:8080"` to the command
-2. Enter "y" to add an Azure cloud provider
-3. Specify your Azure subscription and resource group
-4. Create an [Azure service principal](https://docs.microsoft.com/cli/azure/ad/sp?view=azure-cli-latest#az-ad-sp-create-for-rbac) with the [proper permissions](https://aka.ms/azadsp-more). Enter the appID, password and the tenant of the service principal 
-5. Deploy your app and any included Azure resources with `rad deploy`
+1. Create an [Azure service principal](https://docs.microsoft.com/cli/azure/ad/sp?view=azure-cli-latest#az-ad-sp-create-for-rbac) with the [proper permissions](https://aka.ms/azadsp-more).
```
Comment (AaronCrawfis) [inline docs/content/operations/providers/index.md]:
```suggestion
1. Create an [Azure service principal](https://learn.microsoft.com/cli/azure/ad/sp?view=azure-cli-latest#az-ad-sp-create-for-rbac) with the [proper permissions](https://aka.ms/azadsp-more). 
```

File: docs/content/operations/providers/index.md [hunk 5]
```diff
@@ -36,76 +36,35 @@ The Azure provider allows you to deploy and connect to Azure resources from a se
...
+1. Create an [Azure service principal](https://docs.microsoft.com/cli/azure/ad/sp?view=azure-cli-latest#az-ad-sp-create-for-rbac) with the [proper permissions](https://aka.ms/azadsp-more). 
 
-#### Add a cloud provider to an AKS cluster using AAD pod identity
-
-[Azure Active Directory (Azure AD) pod-managed identities](https://docs.microsoft.com/azure/aks/use-azure-ad-pod-identity) use Kubernetes primitives to associate managed identities for Azure resources and identities in Azure AD with pods. Administrators create identities and bindings as Kubernetes primitives that allow pods to access Azure resources that rely on Azure AD as an identity provider.
-
-1. Enable the preview az aks addon:
    ```bash
-   az extension add --name aks-preview && az extension update --name aks-preview
+   az ad sp create-for-rbac --role Owner --scope --scope /subscriptions/<subscriptionid>/resourceGroups/<resourcegroupname>
```
Comment (AaronCrawfis) [inline docs/content/operations/providers/index.md]:
```suggestion
   az ad sp create-for-rbac --role Owner --scope /subscriptions/<subscriptionid>/resourceGroups/<resourcegroupname> 
```

File: docs/content/operations/providers/index.md [hunk 6]
```diff
@@ -36,76 +36,35 @@ The Azure provider allows you to deploy and connect to Azure resources from a se
...
-
-[Azure Active Directory (Azure AD) pod-managed identities](https://docs.microsoft.com/azure/aks/use-azure-ad-pod-identity) use Kubernetes primitives to associate managed identities for Azure resources and identities in Azure AD with pods. Administrators create identities and bindings as Kubernetes primitives that allow pods to access Azure resources that rely on Azure AD as an identity provider.
-
-1. Enable the preview az aks addon:
    ```bash
-   az extension add --name aks-preview && az extension update --name aks-preview
+   az ad sp create-for-rbac --role Owner --scope --scope /subscriptions/<subscriptionid>/resourceGroups/<resourcegroupname> 
    ```
-1. Deploy an AKS cluster with [pod identity](https://docs.microsoft.com/azure/aks/use-azure-ad-pod-identity) enabled, or enable it on your existing cluster
+   Replace it with your subscription id and resource group name
```
Comment (AaronCrawfis) [inline docs/content/operations/providers/index.md]:
```suggestion
   Make sure to update the command with your subscription id and resource group name
```

File: docs/content/operations/providers/index.md [hunk 7]
```diff
@@ -36,76 +36,35 @@ The Azure provider allows you to deploy and connect to Azure resources from a se
...
-   az aks pod-identity add --resource-group ${MY_RESOURCE_GROUP} --cluster-name ${MY_CLUSTER} --namespace radius-system  --name radius --identity-resource-id ${IDENTITY_RESOURCE_ID}
-   ```
-1. Install the [Radius Helm chart]({{< ref kubernetes >}}) with the azure provider values set:
-   ```bash
-   helm upgrade radius radius/radius --install --create-namespace --namespace radius-system --version {{< param chart_version >}} --wait --timeout 15m0s --set rp.provider.azure.podidentity=radius --set rp.provider.azure.subscriptionId=${MY_SUBSCIRPTION_ID} --set rp.provider.azure.resourceGroup=${MY_RESOURCE_GROUP}
-   ```
-1. Create a new environment:
+   Replace it with your service principal appId, password and tenant id
+
+1. Update your environment with your Azure subscription and resource group
```
Comment (AaronCrawfis) [inline docs/content/operations/providers/index.md]:
```suggestion
1. Update your environment with your Azure subscription and resource group. This is where Azure resources will be deployed.
```

File: docs/content/operations/providers/index.md [hunk 8]
```diff
@@ -120,9 +79,9 @@ The AWS provider allows you to deploy and connect to AWS resources from a Radius
 
 #### Add a cloud provider when initializing an environment
 
-1. Initialize a new [environment]({{< ref environments >}}) with `rad env init kubernetes -i`
-1. Enter "n" to add an Azure cloud provider
-1. Enter "y" to add an AWS cloud provider
+1. Initialize a new [environment]({{< ref environments >}}) with `rad init`
+1. Select the kubernetes cluster to install Radius. Enter an environment name and namespace to deploy the apps into. Note that this namespace is used for application deployments.
```
Comment (AaronCrawfis) [inline docs/content/operations/providers/index.md]:
```suggestion
1. Select the Kubernetes cluster to install Radius into. Enter an environment name and base Kubernetes namespace to deploy the apps into.
```

**Review by AaronCrawfis — CHANGES_REQUESTED** [review]

**Review by AaronCrawfis — APPROVED** [review]

**Pull request merged**